service-registry/src/main/java/org/apache/servicecomb/serviceregistry/api/registry/MicroserviceInstance.java

```diff
@@ -210,4 +211,8 @@ private static void loadDataCenterInfo(MicroserviceInstance microserviceInstance
       microserviceInstance.setDataCenterInfo(dataCenterInfo);
     }
   }
+
+  public boolean isNil() {
```

Comment:
nil keyword is used in other languages like go, maybe isEmpty, isNull or even isExists is more commonly used in Java